lib/pet_store/accounts.ex

```diff
@@ -85,6 +97,25 @@ defmodule PetStore.Accounts do
     |> Repo.insert()
   end
 
+  @doc """
+  Like register user, but should be preferred when registering from the web interface as
+  it checks for adequate admin_level
```

Comment:
If it's "like register user", does it make sense to have them both? Technically now `register_user` is skipping some business logic checks, so I'd expect this to become the new `register_user`, given that when called with `creator == nil` it should have the same effect

Reply:
It technically is, as explained in the commit message (80086e3) `reigster_user` is there only as a shortcut for the tests.
For this reason I should've probably put it in the `accounts_fixture` file instead of the project api, will look into it in a bit to see if I can avoid it altogether

Reply:
Ok so with e8920286cb2cf13385e62f5cb32dcc73e4de8140 I can pass all the tests in the latest branch :+1: